data/os/CentOS-6.yaml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I put it there cause it will be different when we implement TLS.  I probably won't get there this round, do you think I should leave it or copy it up?